omnisharp.json

```diff
@@ -3,6 +3,7 @@
         "enableEditorConfigSupport": true
     },
     "RoslynExtensionsOptions": {
-        "enableAnalyzersSupport": true
+        "enableAnalyzersSupport": true,
+        "enableDecompilationSupport": true
```

Comment:
Thanks, this looks useful - although I'm curious why it's an option for the repository instead of in a local config file for the user?

Reply:
it is a local setting too, just like the other settings you enable via omnisharp.json (analyzers and editorconfig) already 

since you have the file in the repo - and the reason for that is to provide the most optimal editor experience - I thought it would make sense to leverage it to enhance the contributor experience even further 🙂

Reply:
OK, that makes sense. :)